README.md

```diff
@@ -97,8 +97,24 @@ On first launch, the container bootstraps the installation with composer then af
 
 When you see the line `Starting service: openresty` you can navigate to: [https://www.planet4.test](https://www.planet4.test).
 
+**Troubleshooting**
+
 If at any point the install process fails, with Compose showing a message such as `file could not be downloaded (HTTP/1.1 404 Not Found)`, this is a transient network error and re-running the install should fix the issue.
 
+In case the problem persists, try uncommenting these lines in `docker-compose.yml`: 
```

Comment:
For now, until we implement semver promotion on base-fork, I suggest simply uncommenting these lines in the docker-compose file, or will always fail.

Reply:
By which I mean let's actually do that instead of document workarounds: See #8

Reply:
Will do! Should we apply the same principle to the DB schema issue? I think it's probably better to make sure the schemas match and the default content SQL works instead of documenting a way to destroy and recreate the db. 🤔

Reply:
I don't know about the process you undertook, what failed or why, so I can't offer suggestions as to that.  Certainly there's value in automating the import of content in some way, maybe room for one or more tickets here.

Until further investigation, documented as a workaround LGTM